src/richie/apps/core/api.py

```diff
@@ -0,0 +1,45 @@
+"""
+API endpoints for the core app.
+"""
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from rest_framework.viewsets import ViewSet
+
+from .serializers import UserSerializer
+
+
+class UsersViewSet(ViewSet):
+    """
+    API endpoints to access users.
+    """
+
+    # pylint: disable=no-self-use,unused-argument
+    def list(self, request, version):
+        """
+        List requests for users. We do not provide an API to get a list of registered users at
+        this time, and therefore return 501 Not Implemented.
+        """
+        return Response(status=501)
```

Comment:
Did we plan to implement them in the future ? If we don;t know we should probably return a 404